src/corona_hakab_model/infection.py

```diff
@@ -36,8 +36,9 @@ def _perform_infection(self) -> List[Agent]:
         # v = [True if an agent can infect other agents in this time step]
         v = np.random.random(len(self.manager.agents)) < self.manager.contagiousness_vector
```

Comment:
v should be an array floats, maybe remove the random comparison?

Reply:
good point. fixed.
maybe later contagiousness_vector should be renamed to V.